asserts/device_asserts.go

```diff
@@ -81,6 +81,11 @@ func (mod *Model) Kernel() string {
 	return mod.HeaderString("kernel")
 }
 
+// Base returns the base snap the model uses.
+func (mod *Model) Base() string {
```

Comment:
I think we should prevent to set base if the model is classic.  If understand the snapd snap will not have a base, so a classic system doesn't have a base, it will just fetch the bases needed for the snaps.